src/contracts/protocol/libraries/types/DataTypes.sol

```diff
@@ -50,8 +50,9 @@ library DataTypes {
     uint128 variableBorrowIndex;
     //the current variable borrow rate. Expressed in ray
     uint128 currentVariableBorrowRate;
-    // DEPRECATED on v3.2.0
-    uint128 __deprecatedStableBorrowRate;
+    /// @notice reused `__deprecatedStableBorrowRate` storage from pre 3.2
```

Comment:
this is more a `@dev` than a `@notice` imho
how do we know that starts with 0 value?

Reply:
we checked, but also on the upgrade proposal we're iterating all reserves and ensure it's 0.